### PR TITLE
DOCSP-18436: Added correct product names to snooty

### PR DESCRIPTION
**Pull request by DBirtolo-mdb**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-kafka-connector/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-18436
Staging - https://docs-mongodbcom-staging.corp.mongodb.com/kafka-connector/docsworker-xlarge/DOCSP-18436/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [ ] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [x] Are all the links working?

### CODE REVIEW

File: source/introduction/converters.txt [hunk 1]
```diff
@@ -48,8 +48,9 @@ To learn what converter to use, `see this page from Confluent <https://docs.conf
 Converters with Schemas
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-If you use a schema-based converter such as the converter for Avro, Protobuf, or
-JSON Schema, you should define a schema in your source connector. 
+If you use a schema-based converter such as the {+avro-converter+},
+{+protobuf-converter+}, or {+json-schema-converter+} you should define a schema
```
Comment (mongoKart) [inline source/introduction/converters.txt:52]: I: I think there should be a comma at the end of the list

```suggestion
{+protobuf-converter+}, or {+json-schema-converter+}, you should define a schema
```

File: source/source-connector/usage-examples/schema.txt [hunk 2]
```diff
@@ -140,7 +140,7 @@ Your custom schema connector configuration should resemble the following:
 
 .. note:: Embedded Schema
 
-   In the preceding configuration, the JSON Schema converter embeds the custom
+   In the preceding configuration, the {+json-schema-converter+} embeds the custom
    schema in your messages. To learn more about the JSON Schema converter, see the
```
Comment (mongoKart) [inline source/source-connector/usage-examples/schema.txt:144]: Q: Are we only capitalizing "converter" when using the full product name? There are several places on this page, like this one, where it isn't capitalized.

**Review by mongoKart — APPROVED** [review]
LGTM + 1 comma + 1 question

File: source/source-connector/usage-examples/schema.txt (hunk 2 above)
Comment (DBirtolo-mdb) [reply source/source-connector/usage-examples/schema.txt:144]: Correct - the official recommendation was to only capitalize it during the full product name. I opted to leave JSON Schema in for clarity purposes.

**Pull request merged**